alphastats/gui/utils/analysis_helper.py

```diff
@@ -75,12 +93,18 @@ def get_unique_values_from_column(column):
 def st_general(method_dict):
     chosen_parameter_dict = {}
 
-    if "settings" in list(method_dict.keys()):
-        settings_dict = method_dict.get("settings")
+    if "settings" not in method_dict:
+        # TODO: is this check really required here? If so, refactor to be part of plotting/statistic_options
```

Comment:
I am confused by this, too. Has this resolved in the meantime?

Reply:
yes, this logic is now explicit